src/main/proto/ga4gh/rna_quantification.proto

```diff
@@ -22,6 +23,22 @@ enum ExpressionUnit {
   TPM = 2;
 }
 
+// A collection of associated RNAQuantifications.  Typically this will be all
+// the Quantifications of samples from an experiment.
+message RnaQuantificationSet {
+  // The RNA quantification set ID.
+  string id = 1;
+
+  // The ID of the dataset this RNA Quantification set belongs to.
+  string dataset_id = 2;
+
+  // The RNA quantification set name.
+  string name = 3;
+
+  // The RNA quantifications in this set.
+  repeated RnaQuantification rna_quantifications = 4;
```

Comment:
Are you sure you want these here, as opposed to storing the `rna_quantification_set_id` in each `RnaQuantification` object? Otherwise, this object may become unbounded in size.